### PR TITLE
Benji check collector s3 workflow

### LINKED ISSUE #29: create alert for when transit-collector breaks
Right now the transit-collector is running on @sidetrackedmind 's raspberry pi. It was silently broken for 1+ week. this is not great because we cannot get the data back. We need to come up with an alert mechanism in the interim and ultimately move this process off the raspberry pi and into the cloud where alerting / error catching is available to the whole team.

### PR DESCRIPTION
**Pull request by sidetrackedmind**
Fixes #29 

## Proposed changes
The purpose of this pull request is check if opentransit-collector is working properly. When the collector is working properly, it adds a file to S3 () every 15 seconds. The goal of this PR is scheduling a github action to check S3 for the last modified object time and compare that to the current time. In theory, if everything is working, the last modified time should be +- 15 seconds from the current datetime. This action will fail if the difference in time is greater than or equal to 5 minutes.

## Current methodology

1. clone repo
2. download awscli
3. configure awscli
4. check s3 for latest object based on current date (there are a lot of objects in s3, if we do not specify the date it takes a long time to return the latest object) - return last modified datetime of latest object
5. compare last modified datetime to current datetime in seconds
6. conditional - fail if difference is greater than or equal to 5 minutes

## Thoughts on current methodology

I wasn't sure if there's a better way to install `awscli`. Steps 1-3 above seem to be required to have access to awscli. If there's a quicker/shorter way to do it, I would be happy removing/consolidating some steps. Even though the s3 bucket is publicly available, it seems like you have to configure awscli to run it.

### CODE REVIEW

File: .github/workflows/check_s3.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+    - id: repo-checkout
+      name: Check out this repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - id: install-aws
+      name: Install awscli
+      run: |
+        sh awscli_install.sh
```
Comment (liamphmurphy) [inline .github/workflows/check_s3.yml:20]: I would look into using an Action on the GitHub Marketplace for installing the cli, this looks promising https://github.com/marketplace/actions/install-aws-cli-action

One benefit of actions in general is that you can `use` actions made by the community to install various tools. If that works than the bash script wouldn't be needed.

File: backend/compute_arrivals.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from datetime import datetime, date, timedelta
```
Comment (liamphmurphy) [inline backend/compute_arrivals.py:3]: Just want to double check, did you mean to commit all this python code?

**Review by liamphmurphy — APPROVED** [review]
This looks good, I would just look into that Action to download the CLI instead of having to do it manually. 

We could also look into Slack notifications, and again we can use the Actions community for this: https://github.com/marketplace/actions/slack-notify I'm also happy to work on this in a follow-up PR if you'd rather get this workflow out and running.

I'll go ahead and approve and let you determine if you want to work on those two items or not.

File: backend/compute_arrivals.py (hunk 2 above)
Comment (sidetrackedmind) [reply backend/compute_arrivals.py:3]: ah 🤦 . These are leftover updates from the "download arrival data" branch.

**Comment by sidetrackedmind** [comment]
closing PR. Starting another one without the python scripts.

**Pull request closed without merging**